python-toolbox/setup.py

```diff
@@ -91,9 +93,9 @@
     'pytest>=2.9.2',
     'pytest-cov>=1.8.1,<2.6',
     'pytest-watch>=4.1.0',
-    'pytest-testmon>=0.8.2',
+    'pytest-testmon==0.8.2',
     'Keras>=2.2.0',
-    'tensorflow>=1.8.0',
+    'tensorflow==2.0',
```

Comment:
Is upgrading the version of tensorflow something required to allow data sample import from big query? If not I recommend deal with it as a separate commit/MR, just so that it's easy for other users to find in which version we introduced compatibility break to old version of 3rd party frameworks like tensorflow. Other than that the MR lgtm

Reply:
I agree, the dependency setting was made to fix the keras_serializer test on travis. How do I proceed? Should I create another pull request to set the dependencies version and then merge this one?

Reply:
That will be nice @cardosolucas, you can make a PR, and we will merge it first very fast.

Reply:
#38 merged